openwisp_radius/receivers.py

```diff
@@ -44,3 +46,12 @@ def organization_post_save(instance, **kwargs):
         rg.name = rg.name.replace(instance.__old_slug, instance.slug)
         rg.full_clean()
         rg.save()
+
+
+def radiustoken_post_save(instance, **kwargs):
+    username = f'rt-{instance.user.username}'
```

Comment:
this could be a method in the `RadiusToken` model, which would make it consistent with `delete()`